.gitignore

```
@@ -1,5 +1,16 @@
 .idea
+.DS_Store
 
 # Ruby gem
 ruby/*.gem
 ruby/.rspec_status
+
+# Python
+
+python/bin
+python/lib
+python/dist
+python/.Python
+python/include
+python/build
+*.egg-info
```

```suggestion
python/*.egg-info

```
All python specific + newline eof